documentation/modules/mirrormaker2/proc-mirrormaker-replication.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// assembly-mirrormaker.adoc
+
+[id='proc-mirrormaker-replication-{context}']
+= Synchronizing data between Kafka clusters using MirrorMaker 2.0
+
+Use MirrorMaker 2.0 to synchronize data between Kafka clusters through configuration.
+
+The previous version of MirrorMaker continues to be supported.
+If you wish to use the resources configured for the previous version,
+they must be updated to the format supported by MirrorMaker 2.0.
+
+The configuration must specify:
+
+* Each Kafka cluster
+* Connection information for each cluster, including TLS authentication
+* The replication flow and direction
+** Cluster to cluster
+** Topic to topic
+
+Use the properties of the `KafkaMirrorMaker2` resource to configure your Kafka MirrorMaker 2.0 deployment.
+
+MirrorMaker 2.0 provides default configuration values for properties such as replication factors.
+A minimal configuration, with defaults left unchanged, would be something like this example:
+
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
```

Comment:
What does {KafkaApiVersion} resolve to? At the moment, the MM2 CR is only in the alpha API version (`kafka.strimzi.io/v1alpha1`).

Reply:
That's beta1, so I've replaced it with {KafkaApiVersionPrev}